### DISCUSSION TITLE
Citation for academic work

### DISCUSSION THREAD

**Discussion by ckrapu** [google/s2geometry#344] [Q&A]
What would be the best way to cite this work? Currently, I was planning to do this:

S2 Developers. “S2 Geometry Library,” http://s2geometry.io/. Accessed January 14, 2024.

Is there a more appropriate article or set of authors to list?

**Comment by smcallis** [comment]
I found a bit of prior art that uses the Chromium project instead:
https://style.mla.org/citing-source-code/, so I think something like that
would be fine with "S2 Authors" instead.  Otherwise no strong opinions from
me :)
